book/src/bridge/extern_rustqt.md

```diff
@@ -131,7 +131,19 @@ Where `<Property>` is the name of the property.
 
 These setters and getters assure that the changed signal is emitted every time the property is edited.
 
-> Note that in the future it will be possible to specify custom getters and setters
+It is also possible to specify custom getters, setters and on-changed functions, using flags passed like so:
+`#[qproperty(TYPE, NAME, read = myGetter, write = mySetter, notify = myOnChanged)]`
+
+> Note: currently TypeName lookups are not currently supported, so either the name must be camel case or specified like `#[cxx_name = "my_getter"]`
+
+It is also possible to use any combination of custom functions or omitting entirely, but if flags are specified, read must be one of them as all properties need to be able to be read.
```

Comment:
Please also describe what exactly `read` vs `read=my_getter` does.
I.e. if you use `read`, it will auto-generate an appropriate getter with the name of the property and assumes that a member of the same name existst in the Rust struct.
Whereas with `read=my_getter` nothing will be autogenerated and we expect a function with this exact name and a compatible signature to be

1. Implemented on the qobject::T
2. Exported to C++ within the bridge.

Reply:
Done